modules/components/pages/processors/cohere_chat.adoc

```diff
@@ -0,0 +1,485 @@
+
+=== `schema_registry.tls`
+
+Override system defaults with custom TLS settings.
+
+*Type*: `object`
+
+=== `schema_registry.tls.skip_cert_verify`
+
+Whether to skip server-side certificate verification.
+
+*Type*: `bool`
+
+*Default*: `false`
+
+=== `schema_registry.tls.enable_renegotiation`
+
+Whether to allow the remote server to request renegotiation. Enable this option if you're seeing the error message `local error: 
+tls: no renegotiation`.
+
+*Type*: `bool`
+
+*Default*: `false`
+
+Requires version 3.45.0 or newer
+
+=== `schema_registry.tls.root_cas`
+
+Specify a certificate authority to use (optional). This is a string that represents a certificate chain from the parent trusted root certificate, through possible intermediate signing certificates, to the host certificate.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+```yml
+# Examples
+root_cas: |-
+  -----BEGIN CERTIFICATE-----
+  ...
+  -----END CERTIFICATE-----
+```
+=== `schema_registry.tls.root_cas_file`
+
+Specify the path to a root certificate authority file (optional). This is a file, often with a `.pem` extension, which contains a certificate chain from the parent trusted root certificate, through possible intermediate signing certificates, to the host certificate.certificate.
+
+*Type*: `string`
+
+*Default*: `""`
+
+```yml
+# Examples
+root_cas_file: ./root_cas.pem
+```
+=== `schema_registry.tls.client_certs`
+
+A list of client certificates to use. For each certificate specify values for either the `cert` and `key` fields, or `cert_file` and `key_file` fields.
+
+*Type*: `array`
+
+*Default*: `[]`
+
+```yml
+# Examples
+client_certs:
+  - cert: foo
+    key: bar
+client_certs:
+  - cert_file: ./example.pem
+    key_file: ./example.key
+```
+
+=== `schema_registry.tls.client_certs[].cert`
+
+The plain text certificate to use.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `schema_registry.tls.client_certs[].key`
+
+The plain text certificate key to use.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `schema_registry.tls.client_certs[].cert_file`
+
+The path to the certificate to use.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `schema_registry.tls.client_certs[].key_file`
+
+The path of a certificate key to use.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `schema_registry.tls.client_certs[].password`
+
+The plain text password for when the private key is password encrypted in PKCS#1 or PKCS#8 format. The obsolete `pbeWithMD5AndDES-CBC` algorithm is not supported for the PKCS#8 format. 
+
+WARNING: The `pbeWithMD5AndDES-CBC` algorithm does not authenticate ciphertext, and is vulnerable to padding Oracle attacks which may allow an attacker recover the plain text password.
+
+include::components:partial$secret_warning.adoc[]
+
+*Type*: `string`
+
+*Default*: `""`
+
+```yml
+# Examples
+password: foo
+password: ${KEY_PASSWORD}
+```
+=== `schema_registry.oauth`
```

Comment:
I have reworded these descriptions slightly for more context. Could you check for accuracy?

Reply:
LGTM